src/alfasim_sdk/result_reader/aggregator_constants.py

```diff
@@ -1,5 +1,7 @@
 TRENDS_GROUP_NAME = "trends"
 TRENDS_META_ATTR_NAME = "trends"
+GLOBAL_SENSITIVITY_ANALYSIS_META_ATTR_NAME = "global_sensitivity_analysis"
+GLOBAL_SENSITIVITY_ANALYSIS_GROUP_NAME = "global_sensitivity_analysis"
```

Comment:
Won't the actual reading code be added to alfasim-sdk?
Why add just the constants?

Reply:
Martin i only add this constants because in ```SimFacade``` there are other variables used in there tha was added here, mainly constats that is used to save the results.